### ISSUE TITLE
Method Swizzling Failed in iOS 8+

### ISSUE THREAD

**Issue by liushuaikobe** [blackm00n/AKNumericFormatter#3]
In [UITextField+AKNumericFormatter.m](https://github.com/blackm00n/AKNumericFormatter/blob/master/AKNumericFormatter/AKNumericFormatter/UITextField%2BAKNumericFormatter.m#L21),  the implementation of method `deleteBackward` in UITextField is replaced by [`deleteBackwardSwizzle`](https://github.com/blackm00n/AKNumericFormatter/blob/master/AKNumericFormatter/AKNumericFormatter/UITextField%2BAKNumericFormatter.m#L59). It seems that it doesn't work in iOS8, which results in the following scene：

mask: `****-****-***`
placeHolder: `*`
When the text in a UITextField is "1234-5678-9", I press the backspace button in keyboard, and the expected text is "1234-5678", but actually, the text is "1234-5678-". what's worse, the '-' in the end of the string can't be deleted anyway.

I try to figure it out, but with no results.There's someone may have the same problem in [SO](http://stackoverflow.com/questions/26543233/method-swizzling-doesnt-work-on-ios-8).

**Comment by blackm00n** [comment]
You can manually call UITextField's `alertDeleteBackwards` utility method in delegate:

```
- (BOOL)textField:(UITextField*)textField shouldChangeCharactersInRange:(NSRange)range replacementString:(NSString*)string
{
    if( string.length == 0 ) {
        [textField alertDeleteBackwards];
    }
    return YES;
}
```

Not a good solution, I'll take a look, thank you for reporting.

**Comment by liushuaikobe** [comment]
@blackm00n This is a quick and dirty solution, but actually I want to figure out why could this happen.

**Comment by blackm00n** [comment]
Fixed in iOS 8.3
